src/app/modules/shared/components/filters-tabs/filters-tabs.component.html

```diff
@@ -19,35 +23,18 @@
       </select>
     </div>
     <div class="form-group">
-      <ng-select
-        [items]="malfunctionGroupList"
-        bindLabel="name"
-        placeholder="*Група несправності"
-        notFoundText="Не знайдено"
-        (change)="selectMalfunctionGroupType($event)"
-      ></ng-select>
-      <ng-select
-        [items]="malfunctionSubGroupFilteredList"
-        [(ngModel)]="currentMalfunctionSubgroup"
-        bindLabel="name"
-        placeholder="*Підгрупа несправності"
-        notFoundText="Не знайдено"
-        (change)="selectMalfunctionSubGroupType($event)"
-      ></ng-select>
-      <ng-select
-        [items]="malfunctionFilteredList"
-        [(ngModel)]="currentMalfunction"
-        bindLabel="name"
-        placeholder="*Несправність"
-        notFoundText="Не знайдено"
-        (change)="selectMalfunctionType($event)"
-      ></ng-select>
-    </div>
-    <div>
-      <select class="form-control" width="200" (change)="selectPriority($event.target.value)">
```

Comment:
Why you remove ability to choose a priority?

Reply:
we don't remove it, we move this up